ua/extension_object.go

```diff
@@ -38,9 +65,9 @@ type ExtensionObject struct {
 	Value        interface{}
 }
 
-func NewExtensionObject(value interface{}) *ExtensionObject {
+func NewExtensionObject(value interface{}, typeID *ExpandedNodeID) *ExtensionObject {
```

Comment:
It worth making a comment why and what typeID need to be provided i think

Reply:
No one should be using this function imo